react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.nextCount = 1;
+    this.hotCount = 0;
```

Comment:
We generally store values that change over time in `state`. But you don't need extra properties for counting clicks.